pkg/pgmodel/ingestor/duplicates.go

```diff
@@ -0,0 +1,49 @@
+package ingestor
+
+import (
+	"time"
+
+	"go.uber.org/atomic"
+
+	"github.com/timescale/promscale/pkg/log"
+	"github.com/timescale/promscale/pkg/pgmodel/metrics"
+)
+
+const reportDuplicatesInterval = time.Second * 10
+
+var (
+	effectedMetrics             atomic.Uint64
+	isReportDuplicatesActive    atomic.Bool
```

Comment:
note name change and should also be a sync.Once
```suggestion
	LaunchReporterOnce    sync.Once
```